selfdrive/car/hyundai/interface.py

```diff
@@ -29,7 +29,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
 
     if candidate in CANFD_CAR:
       # detect HDA2 with ADAS Driving ECU
-      if Ecu.adas in [fw.ecu for fw in car_fw]:
+      if 0x50 in fingerprint[6]:
```

Comment:
```suggestion
      if Ecu.adas in [fw.ecu for fw in car_fw]:
```

We verified the car does actually return an adas ecu, we just need to make sure it's consistent (failed the first time without any errors logged)